lib/takeaway.rb

```diff
@@ -0,0 +1,45 @@
+require 'twilio-ruby'
+require_relative 'menu'
+require_relative 'text'
+
+
+class TakeAway
+
+  include Text
+
+  attr_reader :menu, :order, :order_total
+
+  def initialize
+    @menu = Menu.new
+    @order = []
+    @order_total = 0
+  end
+
+  def read_menu
+    @menu.menu_list.each { |key, value| puts "#{key}, £#{value}"}
+  end
+
+  def add_item_to_order(menu_item, quantity=1)
+    quantity.times { |order| @order << menu_item}
+    total(menu_item, quantity)
+  end
+
+  def total_cost
+    #{"Your current order is: #{@order}"}
+    'The total cost to pay is: £' + @order_total.to_s
+  end
+
+  def pay(amount)
+    raise 'Error: amount does not equal total cost!' unless amount == @order_total
```

Comment:
Metrics/LineLength: Line is too long. [82/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/SignalException: Use `fail` instead of `raise` to signal exceptions. (https://github.com/bbatsov/ruby-style-guide#fail-method)